### ISSUE TITLE
Accessibility - current page in sidebar should have aria-current="page"

### ISSUE THREAD

**Issue by kburk1997** [docsifyjs/docsify#2226]
#### Problem or desire

Currently, screen readers such as VoiceOver, JAWS, or NVDA cannot differentiate between selected and non-selected links.

#### Proposal

Use `aria-current="page"` on the selected link, and leave non-selected links as-is

#### Implementation

DOM tree:
![Screenshot of DOM tree with active link highlighted. Active link has aria-current="page"](https://github.com/docsifyjs/docsify/assets/11298269/de61ce74-4236-4b3d-9b69-22fd81e9a0f2)

VoiceOver screen recordings:

Without aria-current:

https://github.com/docsifyjs/docsify/assets/11298269/873fdfe3-6a95-43d5-a417-7d34799b10b0

With aria-current:

https://github.com/docsifyjs/docsify/assets/11298269/87b43a78-8c36-4c63-a71c-9b0f880a1d1a

Reference: [MDN - aria-current](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Attributes/aria-current)

**Comment by jhildenbiddle** [comment]
Thanks for creating the issue and for your patience white waiting for it to be addressed, @kburk1997.

I've created a draft PR here: #2254. In that PR you will find a preview link which will allow you to review the changes:

![CleanShot 2023-10-17 at 08 59 27@2x](https://github.com/docsifyjs/docsify/assets/442527/7e4bf479-b02c-46c3-8e98-a99ffefa1ae5)

Before I mark this PR as ready for review by the maintainers, I would love to get your thoughts and/or feedback.

Thanks!
